_posts/2018-02-02-workflow-process-manager.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Using a process manager with the workflow component
```

Comment:
What is a "process manger"? We want to use a more intriguing title or something that one might google, right?